CONTRIBUTING.md

```diff
@@ -160,174 +204,115 @@ information treat the first 80 characters as a title. Even Github itself does
+All the developers in the team should review [open pull requests](https://github.com/raiden-network/light-client/pulls) frequently.
 
-Don't do this:
+We have a lot of tools that automatically check the quality of the code (eslint, prettier). All these are automatically ran by the CI. Therefore, fixes related to linting are usually not included in reviews.
 
-```html
-<template>
-  <v-container>
-    <NoValidProvider v-if="!providerDetected" />
-    <UserDenied v-else-if="userDenied" />
-    <WalletCore v-else />
-  </v-container>
-</template>
-```
+Additionally, reviewers should **not be nitpicky** about the suggested changes they ask from the author. If something is indeed nitpicky then the reviewer is encouraged to state it beforehand. Example:
 
-The for consistensy you should use:
+> nitpick: I don't really think XYZ makes sense here. If possible it would be nice to have it changed to KLM
 
-```html
-<template>
-  <v-container>
-    <no-valid-provider v-if="!providerDetected" />
-    <user-denied v-else-if="userDenied" />
-    <wallet-core v-else />
-  </v-container>
-</template>
-```
+The author of the pull request can choose to implement the feedback or to ignore it.
 
-##### CSS naming scheme
+Authors should make pull request reviews easier. Make them **as small as possible** .
 
-For CSS naming we try to follow the [BEM](http://getbem.com/) methodology. Along with the naming,
-you should try to scope the styles in their respective components.
+When a reviewer starts a review, he should write a comment in the pull request stating he is doing so. For example:
 
-### Workflow
+> Reviewing this now
 
-When developing a feature, or a bug fix you should always start by writing a
-**test** for it, or by modifying existing tests to test for your feature.
-Once you see that test failing you should implement the feature and confirm
-that all your new tests pass.
+This is to keep track of who is reviewing a pull request and to know about ongoing reviews.
 
-Your addition to the test suite should call into the innermost level possible
-to test your feature/bugfix. In particular, integration tests should be avoided
-in favor of unit tests whenever possible.
+For complicated pull requests, that touch the core of the protocol, at least 2 core developers are recommended to have a look and provide an opinion.
 
-Afterwards you should open a Pull Request from your fork or feature branch
-against master. You will be given feedback from the core developers of raiden
-and you should try to incorporate that feedback into your branch. Once you do
-so and all tests pass your feature/fix will be merged.
+When you have trouble to do a review, it is recommended to clone the branch locally and explore the changes with your editor. Run tests and experiment with the changes, so that you can get a better understanding of the changes and give feedback to the author.
 
-#### Contributing to other people's PRs
+#### Contributing to other pull requests
```

Comment:
Not sure if we should keep this

Reply:
I don't see why we should remove it. While we don't often do it, there were at least a few times I had to actually do it, so it is still useful to be there.

Reply:
Ok cool